nx-X11/programs/Xserver/hw/nxagent/Rootless.c

```diff
@@ -351,14 +351,14 @@ void nxagentRootlessRestack(unsigned long children[], unsigned int nchildren)
 
   for (i = 0; i < ntoplevel; i++)
   {
-    fprintf(stderr, "[%p]\n", toplevel[i]);
+    fprintf(stderr, "nxagentRootlessRestack: [%p]\n", toplevel[i]);
```

Comment:
Just thought it would make sense if you're changing the lines to include the function anyway.